### PR TITLE
Show company on user list

### PR DESCRIPTION
**Pull request by danigargar**
#### Type Of Change 
- [x] Small bug fix
- [ ] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [x] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [x] Commits are split per component (schema, portal/platform, kamusers, agis, ..)
- [x] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

### CODE REVIEW

**Review by ironArt3mis — CHANGES_REQUESTED** [review]
Functional review required

File: web/rest/brand/features/provider/webPortal/getWebPortal.feature [hunk 1]
```diff
@@ -23,7 +23,8 @@ Feature: Retrieve web portals
                   "fileSize": 10,
                   "mimeType": "image/jpeg",
                   "baseName": "client-logo.jpeg"
-              }
+              },
+              "company": null
```
Comment (rbatistadev) [inline web/rest/brand/features/provider/webPortal/getWebPortal.feature:27]: I suggest this should be tested

File: web/rest/brand/features/provider/webPortal/getWebPortal.feature [hunk 2]
```diff
@@ -34,7 +35,8 @@ Feature: Retrieve web portals
                   "fileSize": 10,
                   "mimeType": "image/jpeg",
                   "baseName": "user-logo.jpeg"
-              }
+              },
+              "company": null
```
Comment (rbatistadev) [inline web/rest/brand/features/provider/webPortal/getWebPortal.feature:39]: I suggest this should be tested

File: web/portal/brand/src/entities/WebPortal/WebPortal.tsx [hunk 3]
```diff
@@ -54,7 +54,7 @@ const WebPortal: EntityInterface = {
   path: '/web_portals',
   toStr: (row: WebPortalPropertyList<EntityValue>) => `${row.id}`,
   properties,
-  columns: ['name', 'urlType', 'url', 'logo'],
+  columns: ['name', 'urlType', 'company', 'url', 'logo'],
   acl: {
     ...defaultEntityBehavior.acl,
     iden: 'WebPortals',
```
Comment (rbatistadev) [inline web/portal/brand/src/entities/WebPortal/WebPortal.tsx:60]: Cypress test for this please

**Review by rbatistadev — APPROVED** [review]

**Pull request merged**